libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,275 @@
+.nav-marquee{
+  flex: 1 1 auto;
+  background-color: black;
+}
+
+.nav-marquee.light {
+  color: var(--text-color);
+  background-color: #FFF;
+}
+
+.nav-marquee .content-table{
+  border: 1px solid #E8E8E8;
+  background-color: #FAFAFA;
+}
+
+.nav-marquee .content-table .toc-container {
+  flex: 1 1 auto; 
+  padding: 24px;
+}
+
+.nav-marquee .content-table .toc-title {
+  font-size: var(--type-heading-xs-size);
+  margin: 0;
+  font-weight: bold;
+}
+
+.nav-marquee .toc-list {
+  display: flex;
+  flex-direction: column; 
+  gap: 24px;
+  margin-top: 24px;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.nav-marquee .toc-item {
+  min-width: 200px;
+  display: flex;
+  position: relative; 
+  justify-content: space-between;
+  background-color: #FFF;
+  border: 1px solid #4B4B4B;
+  border-radius: 4px;
+  padding: 24px;
+  cursor: pointer;
+  line-height: 1.5;
+}
+
+.nav-marquee .toc-item:hover {
+  box-shadow: 2px 4px 5px #0000001A;
+  border: 1px solid #1473E6;
+}
+
+.nav-marquee .toc-item:focus-within {
+  border: 1px solid #1473E6;
+  background-color: #F5F9FD;
+}
+
+.nav-marquee .marquee p {
+  margin: 0 0 var(--spacing-s);
+}
+
+.nav-marquee .marquee .foreground .icon-area img{
+  height: 20px;
+  float: left;
+  padding-right: 16px;
+ }
+
+.nav-marquee .toc-link-text {
+  flex: 1 1 auto;
+}
+
+.nav-marquee .toc-item a {
+  color: var(--text-color);
+  outline: none;
+}
+
+.nav-marquee .toc-item a:hover {
+  text-decoration: none;
+}
+
+.nav-marquee .toc-link-text p:last-child {
+  margin-bottom: 0;
+}
+
+.nav-marquee .section-title {
+  font-weight: bold;
+  font-size: var(--type-body-s-size);
+  line-height: 20px;
+}
+
+.nav-marquee .section-description {
+  font-size: 14px;
+}
+
+.nav-marquee .toc-arrow {
+  align-self: center;
+  padding-left: 8px;
+}
+
+.nav-marquee .marquee {
+  min-height: unset;
+}
+
+.nav-marquee .marquee .foreground {
+  width: var(--grid-container-width);
+  margin: 0 auto;
+  padding-top: 56px;
+  padding-bottom: 0px;
+}
+
+.nav-marquee .marquee .foreground .media {
+  width: 100%;
+}
+
+@media screen and (min-width: 600px) {
+  .nav-marquee .content-table .toc-list {
+     flex-flow: row wrap;
+  }
+
+  .nav-marquee .toc-item {
+    flex: 1 1 45%; 
+    padding: 16px;
+  }
+
+  .nav-marquee .marquee .foreground .text h2 {
+    font-size: var(--type-heading-xxl-size);
+    clear: both;
+  }
+
+  .nav-marquee .marquee .foreground .text h3 {
+    font-size: var(--type-heading-m-size);
+    text-transform: none;
+  }
+
+  .nav-marquee .marquee .foreground .media {
+    margin: 0;
+  }
+}
+@media screen and (max-width: 600px) {
```

Comment:
I think the preferred approach is to build your way up:
- styles
- min-width: 600px
- min-width: 1200px
- min-width: 1440px

This particular media query can be removed, all styles should be added before the other media queries

Reply:
Mobile first approach please.